core/hw/sh4/modules/mmu.h

```diff
@@ -59,8 +59,8 @@ static INLINE bool mmu_enabled()
 }
 
 template<bool internal = false>
-u32 mmu_full_lookup(u32 va, const TLB_Entry **entry, u32& rv);
-u32 mmu_instruction_lookup(u32 va, const TLB_Entry **entry, u32& rv);
+u32 mmu_full_lookup(u32 va, const TLB_Entry **tlb_entry_ret, u32& rv);
+u32 mmu_instruction_lookup(u32 va, const TLB_Entry **tlb_entry_ret, u32& rv);
```

Comment:
I would prefer to keep `entry` here. The implementation name doesn't help readability.

Reply:
I reverted these changes as there is a local variable named `entry` in the implementations